### PR TITLE
Fix/storybook build crash

### PR DESCRIPTION
**Pull request by LarsDenBakker**
This skips babel and terser for the storybook prebuilt, since that's already es5 and minified. 

It also runs the builds in sequence instead of in parallel, this is because terser takes up the majority of the build time and it's already multi threaded. Running two builds with terser in parallel is therefore not necessary, and potentially problematic.

I'm hoping this fixes https://github.com/open-wc/open-wc/issues/1203

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -33,7 +33,7 @@ dist
 build-stats.json
 stats.html
 .rpt2_cache
-static-storybook
+storybook-static
```
Comment (Westbrook) [inline .gitignore]: `-static` here but `-preview` elsewhere... is this change related to the other files?

**Comment by Westbrook** [comment]
@LarsDenBakker other than copying files into my `node_modules` is there a better way to confirm this fix works for my build...patient 0 as it were.

File: .gitignore (hunk 1 above)
Comment (LarsDenBakker) [reply .gitignore]: It's unrelated, `storybook-static` is the build output which was never added to the gitignore correctly

**Comment by LarsDenBakker** [comment]
I think that's the best confirmation we can get
